llvm/test/CodeGen/AMDGPU/eliminate-frame-index-v-add-co-u32.mir

```diff
@@ -10,6 +10,63 @@
 # RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx1100 -mattr=+wavefrontsize64 -verify-machineinstrs -run-pass=prologepilog %s -o - | FileCheck -check-prefixes=FLATSCRW64,GFX11 %s
 # RUN: llc -mtriple=amdgcn-amd-amdhsa -mcpu=gfx1200 -mattr=+wavefrontsize64 -verify-machineinstrs -run-pass=prologepilog %s -o - | FileCheck -check-prefixes=FLATSCRW64,GFX12 %s
 
+--- |
+  define void @v_add_co_u32_e32__inline_imm__fi_offset0() #0 { unreachable }
```

Comment:
Ouch, adding nounwind this way in MIR is too painful. Is there really no way to directly set this for MIR?